src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -114,20 +136,26 @@ export function ReceiptModal({
               )}
             </styledEl.Field>
 
-            <styledEl.Field>
-              <FieldLabel label="Filled" tooltip={tooltips.FILLED} />
-              <FilledField order={order} />
-            </styledEl.Field>
-
-            <styledEl.Field>
-              <FieldLabel label="Order surplus" tooltip={tooltips.SURPLUS} />
-              <SurplusField order={order} />
-            </styledEl.Field>
-
-            <styledEl.Field>
-              <FieldLabel label="Fee" tooltip={tooltips.FEE} />
-              <FeeField order={order} />
-            </styledEl.Field>
+            {/*TODO: Currently, we don't have this information for parent TWAP orders*/}
+            {/*The condition should be removed once we have the data*/}
+            {(!twapOrder || isTwapPartOrder) && (
```

Comment:
One way to keep the receipt modal agnostic is to have a prop to toggle these optional fields. What do you think?

Also not mandatory for this PR, just raising like the previous comment

Reply:
100% agree